### DISCUSSION TITLE
Sort Check Output by Error Status (Error, Warn)

### DISCUSSION THREAD

**Discussion by samiam376** [rome/tools#4532] [Suggestions]
If lint rules have been set to warn, any errors should take precedence over them when returning the output of rome check. This is especially useful when there are more errors than the diagnostic limit. Since the errors cannot be viewed without increasing the diagnostic limit.

**Comment by ematipico** [comment]
Makes sense!

**Reply by samiam376** [reply]
I think having a --supress-warnings flag on rome check could also be an easier way to achieve this.

**Reply by samiam376** [reply]
@ematipico do you think this approach makes sense? 

https://github.com/rome/tools/pull/4535
